src/arch/host/include/arch/spinlock.h

```diff
@@ -10,7 +10,7 @@
 #ifndef __ARCH_SPINLOCK_H__
 #define __ARCH_SPINLOCK_H__
 
-typedef struct {
+typedef struct spinlock_s {
```

Comment:
yes, this seems good to me, but please also make these just
```
struct spinlock_s {
...
};
```
without `typedef`. One `typedef` in the common header is enough.

Reply:
Done I think (didn't build test).

Reply:
yep, thanks, looks good now